plugin/trino-memsql/src/test/java/io/trino/plugin/memsql/TestMemSqlTimeColumnMapping.java

```diff
@@ -0,0 +1,40 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.trino.plugin.memsql;
+
+import org.testng.annotations.DataProvider;
+import org.testng.annotations.Test;
+
+import static io.trino.plugin.memsql.MemSqlTimeColumnMapping.parseMemSqlLocalTime;
+
+public class TestMemSqlTimeColumnMapping
```

Comment:
Data type mapping tests are usually done as integration tests, see: https://github.com/trinodb/trino/blob/master/plugin/trino-memsql/src/test/java/io/trino/plugin/memsql/TestMemSqlTypeMapping.java#L80

It helps verify the end to end read/write functions and not just the parsing as done here

Reply:
I like the ability to do both - write testable code, you can unit test it *and* integration test it, at no extra "cost"

Reply:
(As these are now part of MemSqlClient, I've removed the unit tests (which are 3000% faster for developing than spinning up a Docker container to find out if you've a mistake in a REGEX, but I digress...)